inst/bs3compat/_shiny_input.scss

```diff
@@ -31,12 +31,3 @@
   text-shadow: inherit;
   border-width: 0;
 }
-
-.selectize-input.focus {
-  @extend .form-control:focus;
-}
```

Comment:
At least one case where this is wrong:

```r
bs_theme_preview(bs_theme(bootswatch = "slate"))
```

(If you bring focus to `selectInput()`, it's red)